app/assets/javascripts/helpers/api_routes.js

```diff
@@ -19,5 +19,11 @@ class ApiRoutes {
     }
   }
 
+  get commissions() {
+    return {
+      create  : `/commissions/create/`
```

Comment:
This is a `POST` request. Typically a post request simply consists of the route `model_name/`, rather than `model_name/create`. Let's change this to reflect that!